app/scrapers/__init__.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import print_function
+from __future__ import absolute_import
 import os, json, sys
-from google import Google
-from duckduckgo import Duckduckgo
-from bing import Bing
-from yahoo import Yahoo
+from .google import Google
+from .duckduckgo import Duckduckgo
```

Comment:
Even though this is the right way for Python3, can we keep the imports as before? This would make the port much easier, and maybe in a later pull we could change this? I'm not too sure whether this would work or not though,